junifer/api/tests/test_functions.py

```diff
@@ -240,32 +247,37 @@ def test_queue_correct_yaml_config(
         The path to the test directory.
     monkeypatch : pytest.MonkeyPatch
         The monkeypatch object.
+    caplog : pytest.LogCaptureFixture
+        The logcapturefixture object.
```

Comment:
Should you use CamelCase in the docstring here, similar to the DataGrabber issue?

Reply:
Yeah I was debating that and I followed what was there but fair point. I'll take care of that in a separate PR for all test functions.

Reply:
ok makes sense, fine by me!